source/concepts.tex

```diff
@@ -0,0 +1,1249 @@
+%!TEX root = std.tex
+\rSec0[concepts.lib]{Concepts library}
```

Comment:
How much larger do you expect this section to grow? It's currently just 13 pages, which is a bit slim for a top-level clause in the library part of the standard.

Reply:
I suspect library will add more concepts here in the future, though I can't speculate how many. Do you think this would fit better as a subclause somewhere in lib-intro?

Reply:
Well, lib-intro only has meta-level info and no specific header files, so that doesn't quite fit. On the other hand, concepts are meta-level stuff, so those might indeed fit. @zygoloid, what do you think?
(The rough math is 30 top-level clauses vs. 1500 pages = 50 pages per clause.)

Reply:
The intent was to put `<concepts>` early so that concept therein could be defined before appearing in the rest of the library spec. I suppose that's not really critical: we use type traits heavily before they are defined.